### ISSUE TITLE
Can I run the container as a satellite? 

### ISSUE THREAD

**Issue by ArjonBu** [jjethwa/icinga2#167]
I want to setup a satellite inside Kubernetes. Is that possible using this image?

**Comment by bebehei** [comment]
No, it's not possible to run this image as a satellite. You may have a look at bodsch's images.

Alternatively, I've converted this image, but I don't maintain it anymore. IDK how much effort you have to put in to have a working image again. See there: https://github.com/bebehei/icinga2-client-docker

**Comment by bebehei** [comment]
While looking at my image, I've seen some people forked it. Most promising one is from @adamdelman: https://github.com/adamdelman/icinga2-client-docker

Maybe he can tell you more, if it's worth checking out his image.
